### PR TITLE
Updated for Leadership Changes

### PR DESCRIPTION
**Pull request by zehicle**

### CODE REVIEW

**Comment by zehicle** [comment]
I signed it.

**Comment by sarahnovotny** [comment]
@zehicle can you reach out to helpdesk@linuxfoundation.org to see why you're not showing as having signed the CLA?

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The community meeting calendar is available as an [iCal to subscribe to] (https:
...
-| [Cluster Ops](sig-cluster-ops/README.md) | [@zehicle (Rob Hirschfeld, RackN)](https://github.com/zehicle) <br> [@mikedanese (Mike Danese, Google] (https://github.com/mikedanese) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-ops) | [#sig-cluster-ops](https://kubernetes.slack.com/messages/sig-cluster-ops) | Thursdays at 1:00 PM PST on [hangouts](https://plus.google.com/hangouts/_/google.com/sig-cluster-ops)|
+| [Cluster Ops](sig-cluster-ops/README.md) | [@zehicle (Rob Hirschfeld, RackN)](https://github.com/zehicle) <br> [@jdumars (Jason DuMars, Dies] (https://github.com/jdumars) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-ops) | [#sig-cluster-ops](https://kubernetes.slack.com/messages/sig-cluster-ops) | Thursdays at 1:00 PM PST on [hangouts](https://plus.google.com/hangouts/_/google.com/sig-cluster-ops)|
```
Comment (idvoretskyi) [inline README.md]: Not "Dies", but "Deis".

**Comment by zehicle** [comment]
@sarahnovotny when I created the CNCF account as the manager, it assumed I was NOT a also a contributor.  Easy to fix.

**Comment by sarahnovotny** [comment]
thanks for the update!

**Comment by philips** [comment]
conflicts :(

**Comment by philips** [comment]
Closing as I think I merged a dupe and conflicts not resolved in a month.

**Pull request closed without merging**